packages/sdks/react-sdk/src/flows.ts

```diff
@@ -0,0 +1,25 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+export { default as AuthProvider } from './components/AuthProvider';
+export {
+  SignInFlow,
+  SignUpFlow,
+  SignUpOrInFlow,
+} from './components/DefaultFlows';
+export { default as Descope } from './components/Descope';
+export { default as useDescope } from './hooks/useDescope';
+export { default as useSession } from './hooks/useSession';
+export { default as useUser } from './hooks/useUser';
+export {
+  getJwtPermissions,
+  getJwtRoles,
+  refresh,
+  getRefreshToken,
+  getSessionToken,
+  isSessionTokenExpired,
+  isRefreshTokenExpired,
+} from './sdk';
+
+// Export ref to baseHeaders so it can be overridden
+export { baseHeaders } from './constants';
+
+export type { ILogger } from './types';
```

Comment:
imho - having all non-flows here (such as hooks, utils, types etc) is a bit problematic in the long run 

if we would want to add more "chunks" in the future 

few use cases that I can think of - 
- lets say someone want only widgets - they will need also ILogger, and maybe hooks
- maybe someone will want only hooks, and not to import flows


I don't think we should go and do this now ofc, but I would start with a dedicate file `common`/`shared` that all the non flow things will go there if that is possible

Reply:
The internal structure is our call, but what’s important are the entry points we expose

Right now, we have the default entry and a `/flows` entry 
Are you thinking we need more entry points at this stage?

IMO, we should balance between adding optimizations and not overcomplicating things with too many entry points Plus, all widgets need to authenticate first, and hooks don’t really make sense without authentication

Let's have a quick sync on it if needed

Reply:
I just noting that from a file structure perspective perspective, I think its better to put all those things in separate file, thats all